test/git-lfs-test-server-api/main.go

```diff
@@ -147,17 +149,16 @@ func buildTestData() (oidsExist, oidsMissing []TestObject, err error) {
 	defer repo.Cleanup()
 	// just one commit
 	commit := test.CommitInput{CommitterName: "A N Other", CommitterEmail: "noone@somewhere.com"}
-	var totalSize int64
 	for i := 0; i < oidCount; i++ {
 		filename := fmt.Sprintf("file%d.dat", i)
 		sz := int64(rand.Intn(200)) + 50
 		commit.Files = append(commit.Files, &test.FileInput{Filename: filename, Size: sz})
-		totalSize += sz
+		meter.AddEstimate(sz)
```

Comment:
I wrote https://gist.github.com/technoweenie/f5ba7b696fb7eded1473d336e99a80d3 to benchmark atomic.  Clearly a channel is not a good way to optimize `atomic.AddInt64()` calls:

```
~p/atomic-bench $ go test . -bench . -v
testing: warning: no tests to run
BenchmarkChannels-8   	2000000000	         0.18 ns/op
BenchmarkAtomic-8     	2000000000	         0.03 ns/op
PASS
ok  	_/Users/rick/p/atomic-bench	6.014s

~p/atomic-bench $ go test . -bench BenchmarkChannels -v
testing: warning: no tests to run
BenchmarkChannels-8   	2000000000	         0.18 ns/op
PASS
ok  	_/Users/rick/p/atomic-bench	5.436s

~p/atomic-bench $ go test . -bench BenchmarkAtomic -v
testing: warning: no tests to run
BenchmarkAtomic-8   	2000000000	         0.04 ns/op
PASS
ok  	_/Users/rick/p/atomic-bench	0.617s
```

Happy to try other approaches if anyone think it'd be worth it, but `sync/atomic` is fast enough for me :)